Source/Scene/Expression.js

```diff
@@ -65,6 +65,16 @@ define([
                 if (defined(val)) {
                     node = new Node(val);
                 }
+            } else if (call === 'rgba') {
```

Comment:
Just curious, I only looked briefly, but is `rgba` and `hsla` the only ways to define alpha in CSS colors?

Also, I thought alpha goes from 0 to 1 in CSS colors, not 0 to 255.  At least that was the example I saw: http://www.w3schools.com/cssref/css_colors_legal.asp